netlify.toml

```diff
@@ -15,3 +15,87 @@ cache-control = "public, max-age=31536000, immutable"
 for = "/images/*"
 [headers.values]
 cache-control = "public, no-transform, immutable, max-age=604800"
+
+[[redirects]]
+  from = "/"
+  to = "/cu/"
+  status = 302
+  force = true
+  conditions = {Country=["BY","MM","CI","CU","CD","IR","IQ","LR","KP","SD","SY","ZW"]}
+
+[[redirects]]
+  from = "/swap"
+  to = "/cu/swap/"
+  status = 302
+  force = true
+  conditions = {Country=["BY","MM","CI","CU","CD","IR","IQ","LR","KP","SD","SY","ZW"]}
+
+[[redirects]]
+  from = "/liquidity"
+  to = "/cu/liquidity/"
+  status = 302
+  force = true
+  conditions = {Country=["BY","MM","CI","CU","CD","IR","IQ","LR","KP","SD","SY","ZW"]}
+
+[[redirects]]
+  from = "/farms"
+  to = "/cu/farms/"
+  status = 302
+  force = true
+  conditions = {Country=["BY","MM","CI","CU","CD","IR","IQ","LR","KP","SD","SY","ZW"]}
+
+[[redirects]]
+  from = "/pools"
+  to = "/cu/pools/"
+  status = 302
+  force = true
+  conditions = {Country=["BY","MM","CI","CU","CD","IR","IQ","LR","KP","SD","SY","ZW"]}
+
+[[redirects]]
+  from = "/ifo"
+  to = "/cu/ifo/"
+  status = 302
+  force = true
+  conditions = {Country=["BY","MM","CI","CU","CD","IR","IQ","LR","KP","SD","SY","ZW"]}
+
+[[redirects]]
+  from = "/competition"
+  to = "/cu/competition/"
+  status = 302
+  force = true
+  conditions = {Country=["BY","MM","CI","CU","CD","IR","IQ","LR","KP","SD","SY","ZW"]}
+
+[[redirects]]
+  from = "/prediction"
+  to = "/cu/prediction/"
+  status = 302
+  force = true
+  conditions = {Country=["BY","MM","CI","CU","CD","IR","IQ","LR","KP","SD","SY","ZW"]}
+
+[[redirects]]
+  from = "/lottery"
+  to = "/cu/lottery/"
+  status = 302
+  force = true
+  conditions = {Country=["BY","MM","CI","CU","CD","IR","IQ","LR","KP","SD","SY","ZW"]}
+
+[[redirects]]
+  from = "/nfts"
```

Comment:
no need for sub route eg `/nfts/*`?

Reply:
Yeah. We only need to show most common pages. No need to go to the detail.